go.mod

```diff
@@ -1,55 +1,99 @@
 module github.com/containers/image/v5
 
-go 1.16
+go 1.17
 
 require (
-	github.com/14rcole/gopopulate v0.0.0-20180821133914-b175b219e774 // indirect
-	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
 	github.com/BurntSushi/toml v1.1.0
-	github.com/cespare/xxhash/v2 v2.1.2 // indirect
-	github.com/containerd/cgroups v1.0.3 // indirect
 	github.com/containers/libtrust v0.0.0-20200511145503-9c3a6c22cd9a
 	github.com/containers/ocicrypt v1.1.5
-	github.com/containers/storage v1.41.0
+	github.com/containers/storage v1.41.1-0.20220607143333-8951d0153bf6
```

Comment:
Are there plans to change this to a "real" version (i.e. v1.41.2) and then vendor this into Podman v4.1 in time for it's release late next week?

Reply:
That is the goal, but we have limited control on runc release schedule.